packages/create-rspress/package.json

```diff
@@ -24,14 +24,15 @@
   ],
   "scripts": {
     "build": "rslib build",
-    "dev": "rslib build --watch",
+    "dev": "rslib build -w",
     "start": "node ./dist/index.js"
   },
   "dependencies": {
     "create-rstack": "1.0.9"
   },
   "devDependencies": {
     "@rslib/core": "0.1.0",
+    "@microsoft/api-extractor": "^7.48.0",
```

Comment:
no need to install at this package?

Reply:
I removed the rslib can api-e from root package.json